data/agent-goerli.conf

```diff
@@ -2,17 +2,24 @@ log-level = "debug"
 deployment-dir = "/data/deployments/goerli"
 fill-wait-time = 120
 unsafe-fill-time = 600
+poll-period = 5.0
+confirmation-blocks = 0
 
 [account]
 # make sure that the filename matches your keystore file
 path = "/data/account/keystore.json"
 password = ""
 
-[chains.l1]
+[chains.base-chain]
 rpc-url = ""
 
+[chains.goerli]
+rpc-url = ""
+confirmation-blocks = 1
+
 [chains.optimism]
 rpc-url = ""
+poll-period = 5.0
```

Comment:
nitpick: doesnt change anything already defined as default above so we can remove this line.

Reply:
Fred had it in his PR, I think as an example